### PR TITLE
MNT: Add name for demensions as dims to data_key

### PR DESCRIPTION
**Pull request by ke-zhang-rd**
For [issue26](https://github.com/NSLS-II/event-model/issues/26)

### CODE REVIEW

File: event_model/schemas/event_descriptor.json [hunk 1]
```diff
@@ -27,6 +27,13 @@
                         },
                         "description": "The shape of the data.  Null and empty list mean scalar data."
                     },
+                    "dims": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        },
+                        "description": "The names for dimensions of the data. Null and empty list if scalar data"
```
Comment (danielballan) [inline event_model/schemas/event_descriptor.json]: I think "Null _or_ empty list if scalar data" would make more sense than "and" here.

**Comment by ke-zhang-rd** [comment]
Closes#26

**Comment by danielballan** [comment]
:+1:

File: event_model/schemas/event_descriptor.json [hunk 2]
```diff
@@ -27,6 +27,13 @@
                         },
                         "description": "The shape of the data.  Null and empty list mean scalar data."
                     },
+                    "dims": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        },
+                        "description": "The names for dimensions of the data. Null or empty list if scalar data"
```
Comment (tacaswell) [inline event_model/schemas/event_descriptor.json]: Is there a way to express in the schema that if this is non-null this must be the same length as shape?

Comment (danielballan) [reply event_model/schemas/event_descriptor.json]: I could not find one. We can enforce it in the composing tools at least.

**Pull request merged**